tests/governance_history.py

```diff
@@ -79,8 +79,8 @@ def run(args):
         proposals_issued += 1
 
         LOG.info("2/3 members accept the proposal")
-        network.consortium.vote_using_majority(primary, new_member_proposal)
-        votes_issued += 1
+        p = network.consortium.vote_using_majority(primary, new_member_proposal)
+        votes_issued += p.votes_for
```

Comment:
I could be wrong here but should this be:

votes_issued = p.votes_for

Reply:
We are counting the total number of votes we issue, and then what we find in the ledger by comparison, so I believe this is right. In practice we only issue a single set of votes, so it will come out to the same thing, but it's still an accumulation.